### PR TITLE
Update campaign_data.py

### PR DESCRIPTION
**Pull request by Daywhite83**
Added July and August Login Campaign Items

### CODE REVIEW

File: data/campaign_data.py [hunk 1]
```diff
@@ -642,6 +642,158 @@
...
+                monthly_100_items + [
+                    6413,   # Astral Cube
+                    3705,   # Far East Hearth
+                    20713,  # Excalipoor
+					18879,	# Rounsey Wand
+					21107,	# Kyuka Uchiwa
+					27757,	# Bomb Masque
+					25672,	# Snoll Masque
+					21095,	# Heartstopper
+					16120,	# Redeyes
```
Comment (kaincenteno) [inline data/campaign_data.py]: needs indent fix

File: data/campaign_data.py [hunk 2]
```diff
@@ -642,6 +642,158 @@
...
+                ],
+            300: [
+                    10053,	# ♪Bomb
+					10063,	# ♪Xzomit
+					21965,	# Zanmato
+					3732,	# Crimson Chest
+					3743,	# Moogle Bed
+					3748,	# Leafkin Bed
+					4064,   # Rem's Tale Ch.1
```
Comment (kaincenteno) [inline data/campaign_data.py]: Indent fix

File: data/campaign_data.py [hunk 3]
```diff
@@ -642,6 +642,158 @@
...
+					4071,	# Rem's Tale Ch. 8
+					4072,	# Rem's Tale Ch. 9
+					4073,	# Rem's Tale Ch. 10
+					1110,	# Black Beetle Blood
+					844,	# Phoenix Feather
+					836,	# Damascene Cloth
+					837,	# Malboro Fiber
+					1311,	# Oxblood
+					9009,	# Etched Memory
+					],
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

File: data/campaign_data.py [hunk 4]
```diff
@@ -642,6 +642,158 @@
...
+					2964,	# Sobek's Skin
+					2966,	# Bukhis's Wing
+					2967,	# Sedna's Tusk
+					3287,	# Orthrus's Claw
+					3288,	# Dragua's Scale
+					3289,	# Apademak's Horn
+					3292,	# Azdaja's Horn
+					3290,	# Isgebind's Heart
+					3291,	# Alfrad's Fang
+				],
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

File: data/campaign_data.py [hunk 5]
```diff
@@ -642,6 +642,158 @@
...
+					10145,	# Cipher: Rughadjeen
+					10150,	# Cipher: Lhe
+					10151,	# Cipher: Mayakov
+					10155,	# Cipher: Bygid
+					10156,	# Cipher: Mildaurion
+					10161,	# Cipher: Rongelouts
+					10166,	# Cipher: Robel-Akbel
+					10178,	# Cipher: Ullegore
+					10179,	# Cipher: Teodor
+					10183,	# Cipher: Darrcuiln
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

File: data/campaign_data.py [hunk 6]
```diff
@@ -642,6 +642,158 @@
...
+					25774,	# Fancy Gilet
+					25775,	# Fancy Top
+					25838,	# Fancy Trunks
+					25839,	# Fancy Shorts
+					26514,	# Poroggo Fleece
+					3743,	# Moogle Bed
+					3718,	# Ado. Tomatoes
+					10075,	# ♪Red Raptor
+					22153,	# Silver Gun
+					3748,	# Leafkin Bed
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

File: data/campaign_data.py [hunk 7]
```diff
@@ -642,6 +642,158 @@
...
+                    ],
+            500: [
+                    10081,  # ♪Noble Chocobo
+					23803,	# Poroggo Cassock
+					3752,	# Colibri Bed
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

File: data/campaign_data.py [hunk 8]
```diff
@@ -642,6 +642,158 @@
...
+                    ],
+            750: [
+                    3885,	# Meldious Plans
+					3886,	# Timbre Case Kit
+					3887,	# Musichinery Kit
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

File: data/campaign_data.py [hunk 9]
```diff
@@ -642,6 +642,158 @@
...
+                    ] + monthly_750_items,
+            1000:
+                monthly_1000_items + [
+					6486,	# Frayed Sack(Pel)
+					6487,	# Frayed Sack(Fer)
+					6488,	# Frayed Sack(Tau)
+				],
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

**Review by kaincenteno — CHANGES_REQUESTED** [review]
thank you for the commit it's greatly appreciated :D 
Please fix indents (I know its annoying)  and I'll take another peek and see if everything is kosher

**Review by Daywhite83 — COMMENTED** [review]
Hopefully fixed indent issue

File: data/campaign_data.py [hunk 10]
```diff
@@ -642,6 +642,158 @@
...
+		    16120,  # Redeyes
+                ],
+            300: [
+                    10053,  # ♪Bomb
+		    10063,  # ♪Xzomit
+		    21965,  # Zanmato
+	            3732,   # Crimson Chest
+		    3743,   # Moogle Bed
+  		    3748,   # Leafkin Bed
+					4064,    # Rem's Tale Ch.1
```
Comment (kaincenteno) [inline data/campaign_data.py]: indent fix

**Review by kaincenteno — CHANGES_REQUESTED** [review]
last change and i'll merge, won't go and verify if itemID cuz #trust 

but besides that looks good thanks @Daywhite83  😄

**Review by Daywhite83 — COMMENTED** [review]
indent fixed hopefully

**Review by kaincenteno — APPROVED** [review]
just noticed an empty line but it looks good awesome work

**Pull request merged**